node/os.ts

```diff
@@ -306,56 +309,6 @@ export function userInfo(
   notImplemented(SEE_GITHUB_ISSUE);
 }
 
-export const constants = {
```

Comment:
I understand what the intent behind this was, however it's best to throw on unimplemented for `Deno.kill` rather than change Node's behaviour and face incompatibility issues in the future